_includes/head.html

```diff
@@ -26,6 +26,12 @@
 <meta name="msapplication-TileColor" content="#ffffff">
 <meta name="msapplication-TileImage" content="{{ '/media/favicon/144x144.png' | relative_url }}">
 <meta name="theme-color" content="#ffffff">
+<meta name="description" content="An open source voxel game engine. Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server.">
+<meta property="og:url" content="https://www.minetest.net" />
+<meta property="og:type" content="website" />
+<meta property="og:locale" content="en_US" />
+<meta property="og:image" content="https://raw.githubusercontent.com/minetest/minetest.github.io/master/favicon.ico" />
+<meta property="og:description" content="An open source voxel game engine. Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server." />
```

Comment:
are we sure these headers should be added to *every* page in this form?

Reply:
it should not, this should only be added to the homepage

Reply:
Icon and such could stay for every page as long as it uses the correct title.

Reply:
The image should be a .png, not a .ico as well. It's not a fav icon.

Also, the locale should be specified in <html>, the og:locale is pointless

Reply:
I will move these only to the `index.html`. Sounds good?